docs/source/documentation.rst

```diff
@@ -404,21 +404,69 @@ Help
 
 If you experience issues during installation and/or use of AxonDeepSeg, you can post a new issue on the `AxonDeepSeg GitHub issues webpage <https://github.com/neuropoly/axondeepseg/issues>`_. We will reply to you as soon as possible.
 
-Manual correction
+Guide for manual labelling
+===============================================================================
+
+Manual masks for training
+-------------------------------------------------------------------------------
+
+If you want to train your own model, you will need manual masks corresponding to your training images.
+
+* These manual masks must be 8 bits PNG files with 1 channel (256 grayscale) and have the same height abd width as the image.
+* Make sure that they contain only 3 values : 0 (black) for background, 127 (gray) for myelin and 255 (white) for axons and no other intermediate values on strutures edges.
+* Make sure that there is no background pixels (black) on the edges between myelin and axons.
+* Make sure that every structure (background, myelin or axons) contains only the pixel value of that specific structure.
+* Examples of image, good mask and bad mask are provided below:
+
+.. figure:: _static/image_example.png
+    :width: 750px
+    :align: center
+    :height: 500px
+    :alt: Image example
+
+    Image example
+
+.. figure:: _static/good_mask_example.png
+    :width: 750px
+    :align: center
+    :height: 500px
+    :alt: Good mask example
+
+    Good mask example
+
+.. figure:: _static/bad_mask_example.png
+    :width: 750px
+    :align: center
+    :height: 500px
+    :alt: Bad mask example
+    
+    Bad mask example
+
+To create a manual mask for training, you can try one of the following:
+
+* Try segmenting your image with AxonDeepSeg default SEM or TEM models and make manual corrections of the segmentation mask in FSLeyes or GIMP software.
+* Create a new manual mask using GIMP.
+* These options are described in the next section "Manual correction of segmentation masks".
+
+Manual correction of segmentation masks
 -------------------------------------------------------------------------------
 
 If the segmentation with AxonDeepSeg fails or does not give optimal results, you can try one of the following options:
 
-**Option 1: manual correction of the segmentation masks**
+**Option 1: manual correction of the segmentation masks with FSLeyes**
+
+* In FSLeyes, load your image and apply ADS prediction with the default SEM or TEM models.
```

Comment:
"apply ADS prediction" --> no, this was already done in the previous section. Leave it here and change previous section, or change text here

Reply:
Resolved in e241f51